lib/src/config_handler.dart

```diff
@@ -109,6 +116,28 @@ class ConfigHandler {
     }
     configFile.writeAsStringSync(newTool, mode: FileMode.append);
   }
+
+  /// Disables the reporting capabilities if true is passed
+  void enableTelemetry(bool reportingBool) {
```

Comment:
As a discussion point, should we create a `Config` class that abstracts over the file and parsing it? I am concerned that if each method that mutates or reads one of these fields re-implements the parsing, that will not be DRY, and if/when we change things we'll forget to update a place.

Reply:
I completely agree, which is what I am trying to do here with the `ConfigHandler` class. This will be the only class that actually interacts with the file. The `Analytics` class that the client will interact with will delegate all actions around parsing, updating, and checking for validity of the file to this `ConfigHandler` class

Reply:
SGTM